test/functional/services/data_grid.ts

```diff
@@ -40,6 +40,8 @@ export function DataGridProvider({ getService }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
 
   class DataGrid {
+    // This test no longer works in the new data explorer data grid table
+    // since each data grid table cell is now rendered differently
```

Comment:
Can you elaborate a bit more about this? Why should that prevent us from reading the data?

Reply:
This function get the text of each table cell using `cCell.find('.euiDataGridRowCell__truncate').text()`. However, in the new data grid table of data explorer, the data-test-subj of `.euiDataGridRowCell__truncate` no longer exists. 

Also now not every data table cell have the same HTML structure. For example, the timestamp column now has a very nested structure where i can only get the text using `cCell.children().children().children().children().text()`. 

While another column like the `_source` column have a list of <dl> and <dd> elements where we need to traverse through all those elements to get the overall text.

Reply:
But it does seem like except for the `_source` column, all the other table cells can retrieve their text using `cCell.children().children().children().children().text()`